apollo-runtime/src/main/java/com/apollostack/android/GraphqlService.java

```diff
@@ -0,0 +1,14 @@
+package com.apollostack.android;
+
+import android.database.Observable;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** A {@link retrofit2.Retrofit} service for executing GraphQL requests. */
+public interface GraphqlService {
+  // TODO: Replace the wildcard with an interface once apollo-api module is merged.
```

Comment:
So this call will return `Observable<GraphQLQuery>`? But why query, this method should return the `DATA`? 
So maybe it should be smth like this:
```
<DATA> Observable<DATA> execute(GraphQLQuery<DATA> query)
```
`GraphQLQuery` will have method `marshal`, and then we don't need `arguments ` param